docs/migrating/from-create-react-app.md

```diff
@@ -41,7 +41,7 @@ Here's an example `package.json`:
 
 Create React App uses the `public` directory for the [entry HTML file](https://create-react-app.dev/docs/using-the-public-folder), whereas Next.js uses it for static assets. It's possible to add static assets here, but Create React App recommends importing them directly from JavaScript files.
 
-- Move any images, fonts, or other static assets to `public`.
+- Move any images, fonts, or other static assets to `public`. In a Next.js application, this directory is kept at the root of your application.
```

Comment:
I don't understand this additional sentence. What does "kept" mean here?

Reply:
It is not, the CRA does keep at the root of the application. In the original issue for this PR, it was mentioned that the docs were not clear about where the directories should go. I thought, this would be nice to highlight for the developers who might not be familiar or just want a re-confirmation.

I can remove it if you think this is inappropriate.

Reply:
Removed and fixed!